cdap-api/src/main/java/co/cask/cdap/api/flow/FlowletDefinition.java

```diff
@@ -191,6 +203,27 @@ public int getInstances() {
   }
 
   /**
+   * @return Map of Stream name and {@link StreamSpecification} created in this Flowlet.
+   */
+  public Map<String, StreamSpecification> getStreams() {
```

Comment:
Are these methods only needed for old Flow configuration API compatibility? Because when we switched to configurer way, I think these methods shouldn't be needed.